voskcli/transcribe.py

```diff
@@ -161,7 +166,10 @@ def write_webvtt_captions(rec_results):
         paragraph = [line]
         write_captions_paragraph(vtt, paragraph)
 
-    return vtt
+    # calculate average confidence coefficient
+    avg_confidence = float("{:.2f}".format(sum(confidence)/len(confidence)))
```

Comment:
Why take a float, convert is to a string, and then parse it again as a float?

- If you want to return a float, I would just return the one you already have. If you want to format it in your output, you cal always use a format string in the print statement later
- If you want to return a string, don't convert it to a float at the end
- 
Personally, I would use the first option.
```suggestion
    avg_confidence = sum(confidence) / len(confidence)
```

Reply:
Sure, I was trying to limit decimal points because there are like 15 points without formatting. 
Would you prefer me to commit your suggestion or should I do another push with squashed commits to keep the history clean?

Reply:
I think Lars is not trying to say that limiting decimal points for printing is wrong per se, but that you're doing it at a bad point in the code. Since whoever called `write_webvtt_captions(...)` will now get the truncated float, even if they wanted greater precision. If you want to format stuff for printing, it's a good rule of thumb to do it right before printing and only for printing.

Also the whole converting back and forth is kinda superfluous.

Reply:
Ok, got it!

Reply:
I guess that really is a weird way of doing this. Still learning python.